common/src/main/java/io/netty/util/concurrent/NettyBlockHoundIntegration.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.util.concurrent;
+
+import reactor.blockhound.BlockHound;
+import reactor.blockhound.integration.BlockHoundIntegration;
+
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+/**
+ * This class integrates Netty with BlockHound.
+ *
+ * It is public but only because of the ServiceLoader's limitations
+ * and SHOULD NOT be considered a public API.
+ */
+public class NettyBlockHoundIntegration implements BlockHoundIntegration {
```

Comment:
Use the internal package?

Reply:
see the javadoc :( Unfortunately, Java's SPI (`ServiceLoader`) does not support non-public implementations

Reply:
I think @johnou did ask you about move the class to `io.netty.util.internal` package... Also You should mark it as `@UnstableApi`

Reply:
Ah, I see! 👍 ok, will do, thanks :)